components/PortalStatistics.lua

```diff
@@ -0,0 +1,1413 @@
+---
```

Comment:
file name doesn't follow standards, should be `portal_statistics`
also should be inside `statistics` folder (like the count module)

Reply:
done